etc/recipes/lsst-libffi/build.sh

```diff
@@ -0,0 +1,15 @@
+./configure --prefix=$PREFIX || exit 1
+make || exit 1
+make install || exit 1
```

Comment:
I suspect you are correct (but I honestly did not puzzle my way through all of conda-build)
https://github.com/conda/conda-build/blob/master/conda_build/build.py#L461